tests/cursor/cursor-tailable-003.phpt

```diff
@@ -34,12 +34,12 @@ $startTime = microtime(true);
 echo "Awaiting results...\n";
 $it->next();
 printf("Waited for %.6f seconds\n", microtime(true) - $startTime);
-
+// Sometimes the cursor will wait for 0.09999 seconds and sometimes it will wait for 0.1000.
```

Comment:
Did you mean `0.0099` and `0.01`, which is 9.9ms and 10ms respectively? As-is, this comment doesn't agree with the pattern below, as you're always expecting `0.0` and that would conflict with a possible `0.1000`.

Reply:
Yes, that's definitely what I meant, thanks!